core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java

```diff
@@ -129,6 +129,19 @@ private void registerEth2Routes(
       final Optional<SlashingProtection> slashingProtection) {
     final ObjectMapper objectMapper = SigningObjectMapperFactory.createObjectMapper();
 
+    // security handler for keymanager endpoints
```

Comment:
nit: Would be better if adding this was conditional on the key manager api being enabled

Reply:
unfortunately Vertx requires this at parsing time, as soon as it sees an auth'd endpoint in the spec it goes look for the a security handler and crashes if none is found.